pyvsystems/wrapper.py

```diff
@@ -5,9 +5,10 @@
 
 
 class Wrapper(object):
-    def __init__(self, node_host, api_key=''):
+    def __init__(self, node_host, api_key='', timeout=''):
```

Comment:
according to https://requests.readthedocs.io/en/master/user/advanced/#timeouts , the default value of `timeout` value should set to `None`, not empty string.